app/src/main/java/com/lagradost/cloudstream3/ui/download/DownloadFragment.kt

```diff
@@ -247,4 +276,33 @@ class DownloadFragment : Fragment() {
             binding?.downloadStreamButton?.extend()
         }
     }
+
+    // Open local video from files using content provider x safeFile
+    private val videoResultLauncher = registerForActivityResult(
```

Comment:
This is very nested, try doing stuff like `if (result.resultCode != Activity.RESULT_OK) return` to remove unnecessary spacing